src/main/java/com/conveyal/datatools/editor/controllers/api/EditorController.java

```diff
@@ -399,7 +344,7 @@ private String uploadEntityBranding (Request req, Response res) {
         int id = getIdFromRequest(req);
         String url;
         try {
-            url = uploadBranding(req, String.format("%s_%d", classToLowercase, id));
+            url = SparkUtils.uploadMultipartRequestBodyToS3(req, "branding", String.format("%s_%d", classToLowercase, id));
```

Comment:
Is this being uploaded in the same location as before? I think previously the s3 uri was `s3://BUCKET/branding/class_id` and maybe now it's `s3://BUCKET/class_id`?

Reply:
Fixed in 164284c72332e390961ece9bf91e2bf7ff660eb3!